### PR TITLE
[1LP][RFR] Moving manual multi-tenancy test

### PR DESCRIPTION
**Pull request by mnadeem92**
Purpose or Intent
=================

### CODE REVIEW

File: cfme/tests/configure/test_multi_tenancy_manual.py [hunk 1]
```diff
@@ -0,0 +1,438 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
+"""Manual tests"""
+
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [pytest.mark.ignore_stream('5.9', '5.10', 'upstream')]
```
Comment (mshriver) [inline cfme/tests/configure/test_multi_tenancy_manual.py]: Need to collect against at least one stream

File: cfme/tests/configure/test_multi_tenancy_manual.py [hunk 2]
```diff
@@ -0,0 +1,438 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
```
Comment (mshriver) [inline cfme/tests/configure/test_multi_tenancy_manual.py]: Why is this included here? We want to lint all python

File: cfme/tests/configure/test_multi_tenancy_manual.py [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [pytest.mark.ignore_stream('5.9', '5.10', 'upstream')]
+
+
+@pytest.mark.manual
+@test_requirements.cfme_tenancy
+def test_tenant_unique_catalog():
```
Comment (mshriver) [inline cfme/tests/configure/test_multi_tenancy_manual.py]: Lots of opportunity for automation here :)

File: cfme/tests/configure/test_multi_tenancy_manual.py (hunk 1 above)
Comment (mnadeem92) [reply cfme/tests/configure/test_multi_tenancy_manual.py]: All test applicable to both 5.9 and 5.10, so removing both.

File: cfme/tests/configure/test_multi_tenancy_manual.py (hunk 2 above)
Comment (mnadeem92) [reply cfme/tests/configure/test_multi_tenancy_manual.py]: I tried to keep the same syntax as set in test_manual.py, 
Thank you for the suggestion, I have removed it now.

File: cfme/tests/configure/test_multi_tenancy_manual.py (hunk 3 above)
Comment (mnadeem92) [reply cfme/tests/configure/test_multi_tenancy_manual.py]: Yes, automating these test cases is on my TODO list.

File: cfme/tests/configure/test_multi_tenancy_manual.py [hunk 4]
```diff
@@ -0,0 +1,434 @@
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [pytest.mark.ignore_stream("upstream")]
+
+
+@pytest.mark.manual
```
Comment (digitronik) [inline cfme/tests/configure/test_multi_tenancy_manual.py]: Required:
This file is for `manual tests` so better to have this `global marker`

Comment (mnadeem92) [reply cfme/tests/configure/test_multi_tenancy_manual.py]: Done, along with manual, i have marked @test_requirements.cfme_tenancy as well, Cause the file is dedicated for manual tests related to multi-tenancy only.

**Comment by izapolsk** [comment]
@mnadeem92, If I understand your changes right, you are going to move your tests to another file and automate them one by one.  am I right ?
It will entail renaming file and marking all not automated tests as manual ones.
Please remove manual from your file name then.
Please mark every test case as manual.

**Comment by izapolsk** [comment]
let's wait for PRT and Travis results.
